src/contexts/W3iContext/hooks/pushHooks.ts

```diff
@@ -112,5 +118,32 @@ export const usePushState = (w3iProxy: Web3InboxProxy, proxyReady: boolean) => {
     }
   }, [pushClient, refreshPushState])
 
+  useEffect(() => {
+    if (!pushClient) {
+      return noop
+    }
+
+    const pushMessageSub = pushClient.observe('push_message', {
+      next: message => {
+        if (message.params.message.url !== dappOrigin) {
+          return
+        }
+
+        const communicator = new JsCommunicator(emitter)
+        communicator.postToExternalProvider(
+          'dappPushNotification',
```

Comment:
Is this naming based on something that's been agreed? We seem to be mixing and matching camelCase and snake case on these calls too: https://github.com/WalletConnect/web3inbox/blob/main/src/w3iProxy/index.ts#L87

Reply:
it should be underscored, I'll do that